src/components/Table/components/RenderRowOrNestedRow/RenderRowOrNestedRow.tsx

```diff
@@ -72,7 +72,11 @@ const RenderRowWithCells = React.memo(({ checked = false, toggleChecked = () =>
                 {columns[cellCounter]}
               </div>
             )}
-            {isComponentFunctionType(content) ? content({ content, colSpan }) : content}
+            {isComponentFunctionType(content) ? (
+              content({ content, colSpan })
+            ) : (
+              <span>{content}</span>
```

Comment:
Wouldn't it be better to pass an id to this?

Reply:
as i discussed with the QA data test id only needed for the parent wrappers